### ISSUE TITLE
Error Enabling Plugin

### ISSUE THREAD

**Issue by marvinestreet** [cloudamqp/terraform-provider-cloudamqp#45]
I'm getting an error when I attempt to install / enable the rabbitmq_prometheus plugin.
Version: 1.4.1 Darwin (Downloaded)

I'm attempting to install / enable the plugin with this configuration:

```
# Enable the Prometheus plugin for metrics
resource "cloudamqp_plugin" "prometheus_plugin" {
  instance_id = cloudamqp_instance.rabbitmq.id
  name        = "rabbitmq_prometheus"
  enabled     = true
}
```

When I run `terraform apply,` I get the following error:

```
Error: json: cannot unmarshal array into Go value of type map[string]interface {}

  on modules/rabbitmq/main.tf line 27, in resource "cloudamqp_plugin" "prometheus_plugin":
  27: resource "cloudamqp_plugin" "prometheus_plugin" {
```

**Comment by marvinestreet** [comment]
This plugin doesn't work in 3.7

**Comment by tbroden84** [comment]
Correct, it's available from 3.8.0. Should on the other hand present a better error message.
